### ISSUE TITLE
Error: Surface outer skull is not completely inside surface outer skin

### ISSUE THREAD

**Issue by RickLicona** [mne-tools/mne-python#8338]
Dear all,

Regarding the DS117 dataset. When I employ the **conductivity for 3 layers (EEG)**,  I got the next error (subject 006):
![imagen](https://user-images.githubusercontent.com/22995049/94989495-e39ea880-053a-11eb-8968-acae1dfa01dc.png)
![imagen](https://user-images.githubusercontent.com/22995049/94989499-e8635c80-053a-11eb-89dd-847243b98c4e.png)

However, employing the same subject, but with conductivity for 1 layer (MEG), there is not any error:

![imagen](https://user-images.githubusercontent.com/22995049/94989530-282a4400-053b-11eb-99a3-370953b1f140.png)

In the case of the subject 002, there is not problem with **conductivity for 3 layers (EEG)**:

![imagen](https://user-images.githubusercontent.com/22995049/94989598-b7375c00-053b-11eb-8c52-79b0fbdea84a.png)
![imagen](https://user-images.githubusercontent.com/22995049/94989606-bd2d3d00-053b-11eb-87e1-8ffa2ed50d56.png)

I only require to use the EEG data (**I know that the 3-layer is unreliable**). Is there any solution for the first error message? or do you suggest me only take the users such as 002?. The idea is to continue with the forward estimation and the inverse stepts.  

By the way, thanks for your proposal MNE pipeline ;).

**Comment by hoechenberger** [comment]
@RickLicona Can you try running this with MNE 0.21, which was just recently released? Which version of FreeSurfer are you using?

@agramfort Any idea what might be going on here?

**Comment by RickLicona** [comment]
> @RickLicona Can you try running this with MNE 0.21, which was just recently released? Which version of FreeSurfer are you using?
> 
> @agramfort Any idea what might be going on here?

I am using MNE 0.21.0 and FreeSurfer 6.0 (due to problems with the most recent versions)

**Comment by agramfort** [comment]
I think that I recall that for this dataset we only used a single layer BEM
to do source loc on MEG only.

Now I think you have the flash MRI images which should give much better BEM
surfaces.

is that what you tried? maybe @jasmainak remembers more as he did most of
the analysis for this paper.

>

**Comment by RickLicona** [comment]
> I think that I recall that for this dataset we only used a single layer BEM to do source loc on MEG only. Now I think you have the flash MRI images which should give much better BEM surfaces. is that what you tried? maybe @jasmainak remembers more as he did most of the analysis for this paper.
> […](#)

Yes, that is right, only employed a single layer for MEG (here there is not any problem). However, I changed it for 3 layers (EEG) `conductivity = (0.3, 0.006, 0.3)  # for three layer (EEG)`. I want to use only EEG data. For the moment I could generate the 3-layer bem surfaces for the users 2, 14 and 18. For the rest of users, I got the error message: _**Surface outer skull is not completely inside surface outer skin**._ 

There is a comment in the project (12-make_forward.py) that said`# Here we only use 1-layer BEM because the 3-layer is unreliable`.
```
    # Here we only use 1-layer BEM because the 3-layer is unreliable
    fname_bem = op.join(subjects_dir, subject, 'bem',
                        '%s-5120-bem-sol.fif' % subject)
```
 Thus, I guess that for this dataset is more recommendable to use MEG data rather than EEG data. It makes sense because there are problems for generating the 3-layer bem surfaces in the majority of the subjects.

**Comment by agramfort** [comment]
that's right

>

**Comment by jasmainak** [comment]
If you want to use 3 layer BEM you should look at the advice here:

https://mne.tools/stable/overview/faq.html?highlight=mri_watershed#my-watershed-bem-meshes-look-incorrect

and also verify that the BEM looks good by plotting it
